### ISSUE TITLE
User testing feedback

### ISSUE THREAD

**Issue by hshahwan** [FACG5/unicornz#126]
Hi @Ollieblue,
Hope you are doing well,
you said in last time that we talked will you send the copy words for the dashboard.
if you prefer we can discuss here and update all words.

**Comment by Ollieblue** [comment]
Not sure which copy words exactly you mean. I thought I had made the changes here. We'll discuss at 1.

**Comment by hshahwan** [comment]
Good morning Marie,it's okay.
The Team and I are waiting your feedback :))

**Comment by Ollieblue** [comment]
Hi :) here is the link to the initial part of the document https://docs.google.com/spreadsheets/d/14smVy8SJKQC3uLD39kGWbIIx-EeLPgZ7s2d0E71mxck
I need to work on it on Saturday with many more results.

**Comment by hshahwan** [comment]
Hi Team, 
@donia98 @amd711 @ahmad1091 @MarwanElkhoudary check the feedback

**Comment by hshahwan** [comment]
@Ollieblue please, make sure to send us the completely conduct User Testing feedback by Sunday evening.

**Comment by Ollieblue** [comment]
Ok. Here is the di

**Comment by Ollieblue** [comment]
Yes. I’ll send the actual document to you each time it is updated.

**Comment by hshahwan** [comment]
Ok, no problem.
